libs/wingc/src/dtsify/snapshots/declarations.snap

```diff
@@ -172,8 +172,21 @@ class ParentClass extends $stdlib.std.Resource {
       })())
     `;
   }
-  _supportedOps() {
-    return [...super._supportedOps(), "static_method", "foo", "bar", "$inflight_init"];
+  get _onLiftDeps() {
```

Comment:
The `on` prefix is confusing.. since this is a property maybe just call this `_liftDeps` or `_lifts` or `_liftMap` or something along those lines...

Reply:
Renamed to `_liftMap`